packages/superset-ui-core/src/query/buildQueryContext.ts

```diff
@@ -24,5 +26,6 @@ export default function buildQueryContext(
     queries: buildQuery(buildQueryObject(formData, queryFields)),
     result_format: formData.result_format || 'json',
     result_type: formData.result_type || 'full',
+    setDataMask: formData.setDataMask ?? emptyFunction,
```

Comment:
I did it because it always same function so it give us not re-render pure components or recalc `useMemo/useCallback hooks` that get this function if all other data same